content/blog/2025-01-23-centos-connect-and-fosdem.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "AlmaLinux at CentOS Connect and FOSDEM 2025"
+type: blog
+author: 
+    name: "benny Vasquez"
+    bio: "Chair, board of directors"
+    image: /users/benny.jpeg
+date: '2025-01-23'
+images:
+  - /blog-images/2025/fosdemconnect.png
+post:
+    title: "AlmaLinux at CentOS Connect and FOSDEM 2025"
+    image: /blog-images/2025/fosdemconnect.png
+---
+
+
+In just one week we'll be kicking off our time in Brussels as we join both [CentOS Connect](https://connect.centos.org/) and FOSDEM for this year's events, and we are so excited. It always feels special for us this year, because this is the time of year that we celebrate the 4-year anniversary of our very first beta version release. 
```

Comment:
Maybe you missed adding the link to the FOSDEM website?

Reply:
I sure did! Added now. :D